src/pages/home/report/ReportActionItem.tsx

```diff
@@ -102,12 +103,21 @@ type ReportActionItemOnyxProps = {
 
     /** The policy which the user has access to and which the report is tied to */
     policy: OnyxEntry<OnyxTypes.Policy>;
+
+    /** Transaction associated with this report, if any */
+    transaction: OnyxEntry<OnyxTypes.Transaction>;
 };
 
 type ReportActionItemProps = {
     /** Report for this action */
     report: OnyxTypes.Report;
 
+    /** The transaction thread report associated with the report for this action, if any */
+    transactionThreadReport: OnyxEntry<OnyxTypes.Report>;
+
+    /** Array of report actions for the report for this action */
+    reportActions: OnyxTypes.ReportAction[];
```

Comment:
`reportActions` isn't actually unused - we use it to get the `transaction` [here](https://github.com/Expensify/App/pull/36934/files#diff-f0bdc7ab41060e24e6bcdc847f25185b372c5f9a072251fe5d3cae2a27887920R929-R936) but apparently the linter isn't picking up on that despite `reportActions` being included in the `ReportActionItemProps` used by `withOnyx`

Reply:
Asked internally in [slack](https://expensify.slack.com/archives/C03TQ48KC/p1710969256324379) and seems like we've suppressed this error in the past when it's come up for this reason, so gonna do that!